src/pymmcore_plus/model/_pixel_size_config.py

```diff
@@ -59,3 +61,20 @@ def create_from_core(
                 for preset in core.getAvailablePixelSizeConfigs()
             }
         )
+
+    def apply_to_core(
+        self,
+        core: CMMCorePlus,
+        *,
+        exclude: Container[str] = (),
+        on_err: ErrCallback | None = None,
+        apply_properties: bool = False,
+        then_update: bool = True,
+    ) -> None:
+        for config_name, preset in self.presets.items():
+            for dev, prop, val in preset.settings:
+                core.definePixelSizeConfig(config_name, dev, prop, val)
+                core.setPixelSizeUm(config_name, preset.pixel_size_um)
```

Comment:
it looks like this line should probably not be inside the `for ... preset.settings` loop? 

also, it's missing `core.setPixelSizeAffine`